src/NuGet.Core/NuGet.Packaging/Rules/UpholdBuildConventionRule.cs

```diff
@@ -15,82 +16,122 @@ namespace NuGet.Packaging.Rules
     internal class UpholdBuildConventionRule : IPackageRule
     {
         private static ManagedCodeConventions ManagedCodeConventions = new ManagedCodeConventions(new RuntimeGraph());
-        string[] _folders = new string[]
+        private static readonly IReadOnlyList<string> BuildFolders = new string[]
         {
             PackagingConstants.Folders.Build,
             PackagingConstants.Folders.BuildTransitive,
-            PackagingConstants.Folders.BuildCrossTargeting
+            PackagingConstants.Folders.BuildCrossTargeting,
+            "buildMultiTargeting"
```

Comment:
What do you think in creating a new package constant for this string?

Reply:
That's a good question, but I consider it a question for whoever renamed "buildMultiTargeting" to "buildCrossTargeting" (they're an alias of the same functionality). Apparently the plan was to remove "buildMultiTargeting", but for some reason it wasn't, and now it's been supported for such a long time it's too dangerous to remove.

Anyway, this pack validation is not the only thing that uses `PackingConstants.Folders`, and if we add buildMultiTargeting, I don't know if that implies that we need to also update the other classes that use it. That's out of scope for fixing these validation bugs. Plus, adding it as a constant might be a signal that it's something we encourage, whereas we want people to use buildCrossTargeting, and maybe there's a possibility that it adds confusion about what the difference is between the two?